config/jobs/kubernetes-sigs/cluster-api-provider-cloudstack/capi-provider-cloudstack-presumbit.yaml

```diff
@@ -0,0 +1,123 @@
+presets:
+- labels:
+    preset-image-build: "true"
```

Comment:
this name is pretty ambiguous as we have a lot of "image" building jobs that look nothing like this.

can we please name it something more specific?